### DISCUSSION TITLE
Raison d'être

### DISCUSSION THREAD

**Discussion by kironia** [eltikia/openroguelike#8] [Q&A] [answered]
Why does this project exist?

**Comment by Stagyrite** [comment, marked as answer]
The reason for this version of the OpenRoguelike project is to provide:
1. [a codebase](https://github.com/stagyrite/openroguelike/ "Stagyrite/openroguelike: The beginning of something that might end as a roguelike development library. Or not, we will see.") that compiles and runs on Linux,
2. a discussion, e.g., how to add [a town level](https://github.com/eltikia/openroguelike/discussions/5 "Custom level · eltikia/openroguelike · Discussion #5"),
3. a port of [an early version](https://github.com/Stagyrite/openroguelike/tree/dsband "Stagyrite/openroguelike at dsband") to [Nintendo 3DS](https://github.com/Stagyrite/openroguelike/releases/tag/v0.1 "Release 3DS Homebrew · Stagyrite/openroguelike").
